src/MicroMamba.jl

```diff
@@ -17,7 +17,7 @@ end
 
 const STATE = State("", true, "", VersionNumber(0))
 
-const DEFAULT_VERSION = v"0.27.0"
+const DEFAULT_VERSION = tryparse(VersionNumber, get(ENV, "JULIA_MICROMAMBA_VERSION", "0.27.0")) 
```

Comment:
Note that this will use the environment variable when the package is precompiled, a change of the environment variable won't have effect at startup time, in case that's relevant.